### ISSUE TITLE
GUI abort

### ISSUE THREAD

**Issue by EmCNabs** [JPC-AV/JPC_AV_videoQC#82]
Tried testing the GUI today and was met with this error. 

```
av-spex-gui
qt.qpa.plugin: Could not find the Qt platform plugin "cocoa" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

zsh: abort      av-spex-gui
```

Looking into it on my end, but if anyone has thoughts on a fix let me know

**Comment by EmCNabs** [comment]
Fixed it! Blake and I got it working

`av-spex --help` ran normally 
Checked the Qt install and it was the current version: 6.7.3

So ran back through the install steps starting with `conda create -n JPC_AV python=3.10.13` and now it works

**Comment by eddycolloton** [comment]
Did you re-run `activate JPC_AV` after install? I added it to the [README](https://github.com/JPC-AV/JPC_AV_videoQC?tab=readme-ov-file#installation-of-av-spex-scripts) only recently so you may not have seen it. If you did re-run `activate` did that have any effect?

**Comment by EmCNabs** [comment]
Yes, I initially ran `activate JPC_AV` when the issue presented and it did not have any effect

After I did `conda create -n JPC_AV python=3.10.13` , I did all the subsequent steps, including `activate JPC_AV` and it all appears to be in working order now.

**Comment by eddycolloton** [comment]
This issue has been addressed by installation improvements (can use pip install. without the -e flag, or brew install av-spex, or just install the dmg)
